jest.config.ts

```diff
@@ -0,0 +1,13 @@
+import type { Config } from "@jest/types"
+const config: Config.InitialOptions = {
+    verbose: true,
+    transform: {
+        "^.+\\.tsx?$": "ts-jest",
```

Comment:
Do we need this configuration as there aren't any tsx files?

Reply:
`x` is optional in this expression. We need this to transform our `ts` files before running the tests.